app/ports/tests/sample_data/submissions.json

```diff
@@ -0,0 +1,206 @@
+[
+    {
+        "id": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXX1",
+        "os": {
+            "macports_version": "2.5.4",
+            "osx_version": "10.14",
+            "os_arch": "i386",
+            "os_platform": "darwin",
+            "cxx_stdlib": "libc++",
+            "build_arch": "x86_64",
+            "gcc_version": "none",
+            "prefix": "/opt/local",
+            "xcode_version": "10.3"
```

Comment:
We could support `clt_version` as well, but then again, we need tests for when the value is present and when it is not, so that's fine.